RevoValidation/src/FormValidation.swift

```diff
@@ -15,6 +15,11 @@ public struct FormValidation {
             Validation(field: field, rules: rules).displayErrorsAt(errorsLabel)
         )
     }
+
+    public mutating func modifyRulesAndValidate(for field:UITextField, rules:Rules){
```

Comment:
aqui tambe, 

update(rules:Rules, for field:UITextField, validate:Bool = true){
}